src/freertos_drivers/st/Stm32Can.hxx

```diff
@@ -45,6 +45,12 @@
 
 #include "stm32f_hal_conf.hxx"
 
+// Max possible number of CAN ifs across the whole STM32 -- the constructor
```

Comment:
Please turn this into a Doxygen style comment with three slashes.

Reply:
Done.